x/authz/keeper/msg_server_test.go

```diff
@@ -352,19 +352,6 @@ func (suite *TestSuite) TestExec() {
 				return authz.NewMsgExec(grantee, []sdk.Msg{msg})
 			},
 		},
-		{
-			name: "invalid nested msg",
-			malleate: func() authz.MsgExec {
-				return authz.NewMsgExec(grantee, []sdk.Msg{
-					&banktypes.MsgSend{
-						Amount:      sdk.NewCoins(sdk.NewInt64Coin("steak", 2)),
-						FromAddress: "invalid_from_address",
-						ToAddress:   grantee.String(),
-					}})
-			},
-			expErr: true,
-			errMsg: "invalid from address",
-		},
```

Comment:
this case never fails with error `invalid from address` since finding grant takes place before.